homeassistant/components/darksky/weather.py

```diff
@@ -103,6 +103,8 @@ def temperature(self):
     @property
     def temperature_unit(self):
         """Return the unit of measurement."""
+        if self._dark_sky is None:
```

Comment:
It's units that is None not _dark_sky.

Reply:
You're right, took a second look at it just now. That's what I get for taking a whack at this before coffee. Fixing now